spec/roadmapster_wizeline_api_organization_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'spec_helper'
+require 'vcr'
+
+RSpec.describe Roadmapster::Wizeline::Organization do
+
+  before :each do
+    @api_token = ENV['WIZELINE_API_TOKEN']
+    @organizations = Roadmapster::Wizeline::Organization.new(token: @api_token)
+  end
+
+  vcr_options = { cassette_name: 'wizeline/get_organizations' }
+  it 'list all organizations for the current user', vcr: vcr_options do
```

Comment:
should it be

it 'list**s** all organizations....

to be consistent with the get_organization cassette?